PolicyProject1/src/main/java/com/example/demo/controller/PolicyController.java

```diff
@@ -1,33 +1,28 @@
 package com.example.demo.controller;
 
+import com.example.demo.model.request.CreatePolicyRequest;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
-
 import com.example.demo.model.Policy;
 import com.example.demo.service.PolicyService;
 
 @RestController
-
-@RequestMapping("task")
-
-public class PolicyController 
+@RequestMapping("/policy")
+public class PolicyController
 {
 	@Autowired
-	PolicyService pol_ser;
-	
-	
-	@PostMapping(path = "/policy")
-//	@Consumes(MediaType.APPLICATION_JSON)
-	public String createPolicy(@RequestBody Policy policy)
+	PolicyService policyService;
+
+	@PostMapping(path = "/")
+	public ResponseEntity<String> createPolicy(@RequestBody CreatePolicyRequest createPolicyRequest)

```

Comment:
here we have created a method createPolicy in which as a request body we are sending createPolicyRequest entity and return type of this method is ResponseEntity of string type.

Reply:
Please read more about ResponseEntity for better understanding: 
https://www.baeldung.com/spring-response-entity

CreatePolicyRequest is request DTO (Data Transfer Objects) and not core entity!